pydantic/_internal/_std_types_schema.py

```diff
@@ -185,6 +185,7 @@ def uuid_schema(_schema_generator: GenerateSchema, uuid_type: type[UUID]) -> cor
                 ),
             ],
             metadata=metadata,
+            serialization=core_schema.to_string_ser_schema(),
```

Comment:
Is it in the right place?

Reply:
this only affects the strict schema right? I don't know how the serialization schemas work, but I would think it would be better to put on the LaxOrStrictSchema if it's all the same. If not, maybe makes sense to add a test in lax mode? I'm not quite sure what the issue is so this comment may not be helpful, let me know if it makes sense or not.